src/watchmaker/workers/salt.py

```diff
@@ -360,6 +380,78 @@ def _get_failed_states(self, state_ret):
             failed_states = state_ret
         return failed_states
 
+    def _install_pip(self, py_exec):
+        if not py_exec:
```

Comment:
Can remove this check now, since it is a required argument

Reply:
you can still pass in `py_exec=None` but i'm fine to remove

Reply:
Yeah, but then we're in real, intentional user error territory. Can pass _any_ bad value for py_exec (e.g. non-empty, non-string, invalid path to python.exe) to generate all sorts of fun exceptions.